src/placetobe-app/placetobe-campagnes-page.html

```diff
@@ -40,23 +41,36 @@
     <placetobe-route
       route="{{campaignRoute}}"
       pattern="/:campaignId"
-      data="{{routeData}}"
+      data="{{subRouteData}}"
       tail="{{subRoute}}"
     ></placetobe-route>
 
     <!-- Gathers campaign(s) specific information for remaining route, for example post invest status -->
     <placetobe-route
       route="{{subRoute}}"
       pattern="/investeren/:status"
-      data="{{subRouteData}}"
+      data="{{investRouteData}}"
     ></placetobe-route>
 
-    <!-- If routeData indicates that the page is about one campaign and if there is no subRouteData (for example post invest)-->
-    <!--TODO: if route is only /campagnes this will also try to load a single campaing. Can be computed [[!!routeData.campaignId && !subRouteData.status]]-->
-    <template is="dom-if" if="{{!subRouteData.status}}">
-      <placetobe-campaign-page route="[[campaignRoute]]" api-endpoint="[[apiEndpoint]]" campaign-id="{{routeData.campaignId}}"></placetobe-campaign-page>
+    <!-- index page for all campaigns-->
+    <template is="dom-if" if="[[!subRouteData.campaignId]]">
+      <placetobe-campagnes-overview-page api-campaign-data="[[apiCampaignData]]" api-endpoint="[[apiEndpoint]]"></placetobe-campagnes-overview-page>
+    </template>
+
+    <template is="dom-if" if="[[subRouteData.campaignId]]">
+
+      <template is="dom-if" if="[[!investRouteData.status]]">
+        <placetobe-campaign-page route="[[campaignRoute]]" api-endpoint="[[apiEndpoint]]" campaign-id="{{subRouteData.campaignId}}"></placetobe-campaign-page>
+      </template>
+
+      <template is="dom-if" if="[[_computeInvestmentSuccessfull(investRouteData.status)]]">  
```

Comment:
Deze zijn nog leeg. Hier moeten de invest pagina's weer in, anders kom je nergens terecht :)